### PR TITLE
Add CloudFlare

### LINKED ISSUE #31: Cloudflare
**What is the vendor's name?**
Cloudflare
https://blog.cloudflare.com/introducing-single-sign-on-for-the-cloudflare-dashboard/
(It is indicated in the page that SSO is reserved to "Enterprise" customers)

**What is the vendor's pricing site?**
https://www.cloudflare.com/plans/

**What is the base pricing? Use the lowest tier that looks sane for a small business customer, not free or personal tiers.**
200 USD per month

**What is the minimum pricing for SSO support?**
"Customer Success Manager" must be contacted

**Does this pricing info come from a quote or other non-public source?**
Enterprise plan starts at 5000 USD per month
https://support.cloudflare.com/hc/en-us/articles/200170326-How-much-does-the-Enterprise-Plan-cost-

**Are there any caveats we should list in the footnotes?**
It is indicated in the comments of the following page https://blog.cloudflare.com/introducing-single-sign-on-for-the-cloudflare-dashboard/ that they plan to support it for other customers in the future but it seems a very vague statement.

### PR DESCRIPTION
**Pull request by BowlesCR**
Fixes #31

### CODE REVIEW

File: _data/vendors.yml [hunk 1]
```diff
@@ -41,6 +41,14 @@
   pricing_source: https://checklyhq.com/pricing/
   updated_at: 2019-07-30
 
+- name: CloudFlare
+  url: https://www.cloudflare.com
+  base_pricing: $20
```
Comment (Wild1145) [inline _data/vendors.yml:46]: Would suggest that given they have a free offering that the free offering is their "Base Price"

**Review by Wild1145 — APPROVED** [review]
Ignore my previous comment, just spotted something that contradicted my statement. This looks good and should be added.

**Comment by ASutterWork** [comment]
please promote. i am being told i need CF Enterprise to get SSO. Enterprise is much more expensive than the base price.

**Comment by mbainter** [comment]
Given this is now unmaintained, there is now [a fork](stopthessotax/sso-wall-of-shame) that we're hoping to keep maintained via community participation.  If this is still the case with Cloudflare, would you please submit a fresh PR with current pricing details to that repository?

**Comment by robchahin** [comment]
#199 adds a laughable quote price to this, so will use that one

**Comment by abracadabra2** [comment]
Please Re-Add Cloudflare!

**Pull request closed without merging**